recipe/meta.yaml

```diff
@@ -1,43 +1,58 @@
 {% set name = "cmarkgfm" %}
-{% set version = "0.8.0" %}
+{% set version = "0.4.2" %}
```

Comment:
how is this possible? Is this an old versioned feedstock build? I believe in this case we want to follow https://github.com/anaconda-distribution/perseverance-skills/blob/main/sections/02_Package_building/01_How_tos/Creating_versioned_feedstocks_from_existing_packages.md

Reply:
This has been ripped out from something that wasn't versioned on `aggregate` correctly. We were building 0.4.2 in that instance, I've forked for now and can update, however I have not tested `readme_renderer` with 0.8.0 which is what I'm actually dragging from `aggregate` into a submodule

Reply:
We only have cmarkgfm on osx-64. It seems it is only used by readme_renderer, which in turn is only used by twine, which is used to upload to pypi. I don't think we need to update any of these. I suggest to move these 3 feedstocks to aggregate-deprecated instead.

Reply:
After checking in refinement, we should actually support twine. 
In that case twine should be updated to the latest (and cmarkgfm too).

Reply:
I think this point is resolved, right?

Reply:
yes we are building it